apps/frontend/src/pages/project-access-management/libs/components/project-group-users-table/libs/components/project-group-users-search/project-group-users-search.tsx

```diff
@@ -1,17 +1,35 @@
+import {
+	type Control,
+	type FieldErrors,
+	type FieldPath,
+} from "react-hook-form";
+
 import { Search } from "~/libs/components/components.js";
+import { type SearchForm } from "~/libs/types/types.js";
 
 import styles from "./styles.module.css";
 
 type Properties = {
+	control: Control<SearchForm>;
```

Comment:
let's remove SearchForm type and use generic too

Reply:
Now that is fixed.